test/cpp/test_xla_sharding.cpp

```diff
@@ -64,5 +64,81 @@ TEST_F(XLAShardingTest, ShardTensor) {
   EXPECT_EQ(shards[7].sizes(), c10::ArrayRef<long>({8, 7, 4}));
 }
 
+TEST_F(XLAShardingTest, CreateTensorsData) {
+  if (xla::sys_util::GetEnvString(xla::env::kEnvPjRtDevice, "") == "") {
+    GTEST_SKIP() << "`PJRT_DEVICE` is not set.";
```

Comment:
does this work on both PJRT:CPU and PJRT:TPU?

Reply:
Yea, that's the goal.

Reply:
we are trying to merge this pr and will run this test as part of the CI. Would be nice if it works on PJRT:CPU when we merge otherwise we have no way of knowing other pr breaks it

Reply:
Just confirmed, this works on both PJRT:CPU and PJRT:TPU cc @JackCaoG

Reply:
can we make sure tests runs on PJRT:CPU on CI?